app/views/casa_cases/_form.html.erb

```diff
@@ -60,10 +60,11 @@
             <div class="field form-group">
               <%= cdf.text_field :date,
                                  data: {provide: "datepicker", date_format: "yyyy/mm/dd"},
-                                 class: "form-control",
-                                 :required => 'required' %>
+                                 class: "form-control" %>
             </div>
         <% end %>
+        <%= form.check_box :empty_court_date, class: "toggle-court-date-input" %>
+        <%= form.label :empty_court_date, "I don't have it at the moment" %>
```

Comment:
I know the ticket issue gave the "I don't have it at the moment", but I think the "it" is a little confusing to what it is referring to. Can we change it to: 

```suggestion
        <%= form.label :empty_court_date, "I don't know the court date yet" %>
```

Reply:
I'll change it.